libraries/AP_Scripting/lua/src/luac.c

```diff
@@ -435,7 +435,7 @@ static void PrintDebug(const Proto* f)
  printf("upvalues (%d) for %p:\n",n,VOID(f));
  for (i=0; i<n; i++)
  {
-  printf("\t%d\t%s\t%d\t%d\n",
+  printf("\t%d\t%s\t%u\t%u\n",
```

Comment:
Remove the change here, this is external code dependencies, and I'd rather not add conflicts.